sympy/physics/quantum/tests/test_cg.py

```diff
@@ -171,6 +171,7 @@ def test_cg_simp_sum():
 def test_doit():
     assert Wigner3j(1/2, -1/2, 1/2, 1/2, 0, 0).doit() == -sqrt(2)/2
     assert Wigner6j(1, 2, 3, 2, 1, 2).doit() == sqrt(21)/105
+    assert Wigner6j(3, 1, 2, 2, 2, 1).doit() == sqrt(21) / 105
```

Comment:
I assume that the other tests didn't expose this bug and that this new test does. If so, +1.